app/settings.py

```diff
@@ -3,7 +3,7 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-DEBUG = False
+DEBUG = os.environ.get('DEBUG').lower() == 'true'
```

Comment:
I hope the lower() is not called on None in case the env var is not set.

Reply:
Hmm, it may:
```
os.environ.get('DDD').lower()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-cf962eba5d95> in <module>
----> 1 os.environ.get('DDD').lower()

AttributeError: 'NoneType' object has no attribute 'lower'
```
What about adding a default?

Reply:
Oh. Fixed